File: C/Daily Flash/Week1/Day4_Jan16/code_C/program4.c
/*
 * Program 4: Write a Program to Print following Pattern
1 2 3 4
5 6 7 8
9 10 11 12
13 14 15 16
*
*/

#include<stdio.h>

void main() {
	int row;
	printf("Enter num of rows: ");
	scanf("%d",&row);
		
	int count = 1;
	for(int olc = 0; olc<row ;olc++) {
		for(int ilc = 0; ilc<row ;ilc++) {
			if(count < 10) 
				printf(" %d ",count++);
			else
				printf("%d ",count++);
		}
		printf("\n");
	}

}

File: PYTHON/Daily Flash/week1/Day3_Jan15/code_python/program4.py
'''
Program 4: Write a Program to Print following Pattern
1 2 3 4
1 2 3 4
1 2 3 4
1 2 3 4
'''

var = int (input ("num of rows: ") )

for x in range(var):
    for y in range(var):
        print(x+1 ,end =" ")
    print()

